ios/Surveys/Sources/Presentation/Modules/Home/HomeView.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  HomeView.swift
+//  Surveys
+//
+//  Created by Su Ho on 03/11/2022.
+//  Copyright © 2022 Nimble. All rights reserved.
+//
+
+import SwiftUI
+
+struct HomeView: View {
+
+    var body: some View {
+        VStack {
+            HomeHeaderView(
+                title: "Monday, JUNE 16",
+                imageURLString: "https://cdn.pixabay.com/photo/2016/11/18/23/38/child-1837375__340.png"
+            )
```

Comment:
Please add a TODO comment for demo/testing logic 🙏 For example:
- TODO demo data for POW
- TODO integrated by https://github.com/suho/kmm-ic/issues/24

It's an optional point but helps the reviewer so much to get what's happening here
🤷‍♂️